### DISCUSSION TITLE
¿Los nuevos extratypes tendrán el campo de tipo properties?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#66] [2. Questions and Answers] [answered]
Hola
Quisiera saber si está considerado, para los nuevos extratypes (3.0), los tipo de properties (extratypes relacionado al property type)
![image](https://user-images.githubusercontent.com/61419471/114409348-5d61bf80-9b78-11eb-8268-a1286ecc57ee.png)

Revisé y no encontré esa opción. Los extratypes 2.0, que se agregaban por base de datos, contaban con esa opción.

El ejemplo que tengo es: en una ficha de cliente tengo tres campos: **giro comercial**, **comuna** y **región**. Los puedo dejar como subproperty, pero estarán fuera de la ficha cliente (o de los extratypes) y no los podré editar por data base, sino sería por un survey + bot o por web.admin.

Muchas gracias y quedo atento:

PD: Extratype 1.0 son los que se creaban desde el admin-legacy.

**Comment by acedward** [comment, marked as answer]
@NicolasPLG Se llaman **Element**

![image](https://user-images.githubusercontent.com/3951895/114419156-31970780-9b81-11eb-9045-1acf1423720d.png)

**Reply by NicolasPLG** [reply]
gracias @acedward pensé que lo iba a ver como Properties y era elemento.

Gracias
